gobblin-modules/gobblin-couchbase/scripts/install_test_deps.sh

```diff
@@ -10,5 +10,9 @@ if [ -d mock-couchbase/.git ];
     git -C mock-couchbase checkout b929a9f99a31a233c43bd94b6a696ad966206e02
   fi
 pushd mock-couchbase
-mvn package -Dmaven.test.skip=true
```

Comment:
I could see multiple failures possible: 
 - mvn might not be installed
 - mvn could be installed and misconfigured in which case I saw a 'Could not find main class...'  error
 - The downloading of the modules could cause disk to get filled up
 - There could be a transient network issue
... maybe more.

So I thought since we needed the command to fully succeed, we could conditionalize progress on the entire command to return 0.